pyquotes/scrapper.py

```diff
@@ -0,0 +1,44 @@
+from bs4 import BeautifulSoup
+import requests
+import random
+
+# website used :
+parent_link = "https://www.quoteload.com"
+
+
+def get_quotes(person: (None, str) = None, category: (None, str) = None):
+    # function called without any argument:
+    if isinstance(person, type(None)) and isinstance(category, type(None)):
+        print('too few arguments to get quotes')
+        return None  # can be modified to throw some exception instead
+
+    result = []
+    if not isinstance(person, type(None)):  # person's name is provided
```

Comment:
It is better to use `person is not None`

Reply:
Hi, have a look at my commit. Made changes as requested.

Reply:
@akshatkarani updated.